### DISCUSSION TITLE
table size vertipaq analyzer dax studio

### DISCUSSION THREAD

**Discussion by arnohoedelmans** [DaxStudio/DaxStudio#1057] [Q&A]
Hi,

I can't understand following.

In my dataset I copied a table using dax like:
table1 = existing_table

Now i was trying to reduce the data size of table 1 by only selecting needed records and needed columns like:
table1 = summarize(filter(existing_table,column1 = "a"),existing_table[column1],existing_table[column2])

When i look in vertipaq analyzer the table size increases. 

Is this correct? I don't understand and was expecting that it would be less.

**Comment by marcosqlbi** [comment]
Check these articles: 
https://www.sqlbi.com/blog/marco/2022/12/15/fixing-incorrect-dictionary-size-reported-by-vertipaq-analyzer/
https://www.sqlbi.com/blog/marco/2022/04/15/dictionary-size-in-tabular-models-reported-by-vertipaq-analyzer/
https://www.sqlbi.com/articles/measuring-the-dictionary-size-of-a-column-correctly/
